app/controllers/tickets_controller.rb

```diff
@@ -5,7 +5,11 @@ class TicketsController < ApplicationController
   authorize_resource :conference_registrations, class: Registration
   before_filter :check_load_resource, only: :index
 
-  def index; end
+  def index
```

Comment:
I am not sure if I understand what you want to do here.

This index page is shown both when buying registration tickets and when buying other kind of thicket. If you remove the registration tickets from here, where are they being rendered? :confused:

Reply:
I would show them instead, but disabling them. Because this could be a little confusing for the user. But I am fine with doing this in a different PR. :+1:

Reply:
Rather than hiding the tickets I would prefer to "deactivate" the links... but showing them.

Reply:
@mdeniz Yes absolutely a much better fix. 
@Ana06  lets do this in this PR only not many changes required. 😉